validation/consistency_log.go

```diff
@@ -0,0 +1,133 @@
+package main
+
+import (
+	"bytes"
+	"encoding/json"
+	"io/ioutil"
+	"log"
+	"math/rand"
+	"net/http"
+	"os"
+	"strconv"
+	"strings"
+	"time"
+)
+
+const URL = "http://localhost:8090/kv"
+
+var DURATION int64 // in nanosecond
+var CLIENT_ID string
+var EVENT_ID int64
+var KEY string
+
+func checkFatal(e error) {
+	if e != nil {
+		log.Fatal(e)
+	}
+}
+
+func Read() string {
+	start := time.Now().UnixNano()
+	resp, err := http.Get(URL + "?key=" + KEY)
+	checkFatal(err)
+
+	defer resp.Body.Close()
+	end := time.Now().UnixNano()
+
+	body, err := ioutil.ReadAll(resp.Body)
+	checkFatal(err)
+
+	value := strings.Split(strings.Split(string(body), "is ")[1], " with")[0]
+	revision := strings.Split(strings.Split(string(body), "revision ")[1], "\n")[0]
+	//log.Println("read", value, start, end, CLIENT_ID)
+	output := "read," +
+		value + "," +
+		strconv.FormatInt(int64(start), 10) + "," +
```

Comment:
replace 10 to a variable with explanatory name. same for below occurrences.

Reply:
added a constant for this